app/models/drawing.rb

```diff
@@ -2,6 +2,9 @@ class Drawing < ActiveRecord::Base
   enum status: %i(pending complete)
   enum gender: %i(not_specified female male other)
 
+  STAGES_OF_JOURNEY = ["At home", "In temporary shelter", "Awaiting transit", "On the move", "Arrived at destination"].freeze
+  SUBJECT_MATTERS = ["Home/Country of origin", "In transit", "Camp life", "Future_hopes/destination"].freeze
```

Comment:
You'll need to keep the current format of these subject matters to make the data consistent with drawings already uploaded:

```ruby
["Home / Country of origin", "Transit", "Camp life", "Future hopes / destination"]
```